agent/dns_test.go

```diff
@@ -9,13 +9,15 @@ import (
 	"testing"
 	"time"
 
+	require "github.com/stretchr/testify/require"
```

Comment:
nitpick: Probably a tooling thing but there's no need to alias this import - the package is already named "require". goimports does that to me sometimes too and I don't always notice not sure why...